src/components/PostList/PostList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PostList.scss';
+import Post from '../Post/Post';
+
+import users from '../../api/users';
+import comments from '../../api/comments';
+
+function PostList({ posts }) {
+  return (
+    <div className="post-list">
+      {posts.map((post) => {
```

Comment:
if you want to assign value to another variable then `.map()` is not the right method for this, use `.forEach()`